lean4-infoview/src/infoview/index.css

```diff
@@ -179,6 +173,7 @@ select {
 .monaco-hover .markdown-hover > .hover-contents:not(.code-hover-contents) {
 	/* max-width: 500px; */
 	word-wrap: break-word;
+	line-height: 1.5em;
```

Comment:
I am guessing we only want this line height for markdown?

Reply:
right, I copied that from vscode, my main goal was to ensure our tooltips match what you see in the vscode editor.